apps/frontend/src/app/pages/dashboard/community/community-news/community-news.component.ts

```diff
@@ -1,8 +1,55 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { MomBlogService } from '@momentum/frontend/data';
+import { finalize } from 'rxjs/operators';
+import * as txml from 'txml';
```

Comment:
From txml docs (https://www.npmjs.com/package/txml) under Installation:
>For specially small builds using modern module bundlers like rollup or webpack you can import txml/txml or > txml/dist/txml. This will not add the transformStream into the bundle and with that exclude the Node.js files.